artifacthub-pkg.yml

```diff
@@ -1,43 +1,42 @@
----
-version: 0.1.7
+# Kubewarden Artifacthub Package config
+#
+# Use this config to submit the policy to https://artifacthub.io.
+#
+# This config can be saved to its default location with:
+#   kwctl scaffold artifacthub > artifacthub-pkg.yml 
+version: 0.1.7-9-g1fe372d
```

Comment:
I think it is necessary to regenerate the file after changing the command to get the value for the `VERSION`  variable